docs/technical_reference/property_models/coagulation.rst

```diff
@@ -0,0 +1,130 @@
+Coagulation Property Package
+============================
+
+This package implements property relationships for water density as a function of
+temperature, pressure, and mass fraction of suspended/dissolved solids from
+`Engineering Toolbox. (2003) <https://www.engineeringtoolbox.com/water-density-specific-weight-d_595.html>`_
+and water viscosity as a function of temperature from
+`D.S. Viswananth, G. Natarajan. (1989) <https://www.osti.gov/biblio/6562161>`_.
+
+This coagulation property package:
+   * supports only 'H2O', 'TDS', 'TSS', and 'Sludge' as Components

```

Comment:
Consider spelling out "TDS" and "TSS" where they first appear.

Reply:
In this case, these are specific names of components in the actual code, so that should not be changed here as these are the names by which those model components are referenced by. The acronyms are spelled out in the `unit_model` section, but I will add a note below this list for reference.